Cargo.toml

```diff
@@ -132,46 +132,46 @@ revm-primitives = { version = "15.1.0", features = [
   "std",
   "serde-bincode-compat",
 ] }
-reth-primitives-traits = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.5", default-features = false }
-reth-provider = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.5" }
-reth-revm = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.5" }
-reth-rpc = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.5" }
-reth-rpc-api = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.5" }
-reth-rpc-eth-api = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.5" }
-reth-rpc-eth-types = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.5", default-features = false }
-reth-rpc-layer = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.5" }
-reth-rpc-server-types = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.5" }
-reth-rpc-types-compat = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.5" }
-reth-tasks = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.5" }
-reth-transaction-pool = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.5" }
-reth-trie = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.5" }
-reth-trie-common = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.5" }
-reth-trie-db = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.5" }
+reth-primitives-traits = { git = "https://github.com/alpenlabs/reth.git", rev = "e4940d6d09067bad", default-features = false }
+reth-provider = { git = "https://github.com/alpenlabs/reth.git", rev = "e4940d6d09067bad" }
+reth-revm = { git = "https://github.com/alpenlabs/reth.git", rev = "e4940d6d09067bad" }
+reth-rpc = { git = "https://github.com/alpenlabs/reth.git", rev = "e4940d6d09067bad" }
+reth-rpc-api = { git = "https://github.com/alpenlabs/reth.git", rev = "e4940d6d09067bad" }
+reth-rpc-eth-api = { git = "https://github.com/alpenlabs/reth.git", rev = "e4940d6d09067bad" }
+reth-rpc-eth-types = { git = "https://github.com/alpenlabs/reth.git", rev = "e4940d6d09067bad", default-features = false }
+reth-rpc-layer = { git = "https://github.com/alpenlabs/reth.git", rev = "e4940d6d09067bad" }
+reth-rpc-server-types = { git = "https://github.com/alpenlabs/reth.git", rev = "e4940d6d09067bad" }
+reth-rpc-types-compat = { git = "https://github.com/alpenlabs/reth.git", rev = "e4940d6d09067bad" }
+reth-tasks = { git = "https://github.com/alpenlabs/reth.git", rev = "e4940d6d09067bad" }
+reth-transaction-pool = { git = "https://github.com/alpenlabs/reth.git", rev = "e4940d6d09067bad" }
+reth-trie = { git = "https://github.com/alpenlabs/reth.git", rev = "e4940d6d09067bad" }
+reth-trie-common = { git = "https://github.com/alpenlabs/reth.git", rev = "e4940d6d09067bad" }
+reth-trie-db = { git = "https://github.com/alpenlabs/reth.git", rev = "e4940d6d09067bad" }
```

Comment:
If this gets through. Can we try to upstream this patch to `reth`?
What was the change that you had to do? I could not find in the PR description.

Reply:
Amazing! Music to my ears. Cc @MdTeach

Reply:
Can you please add

> This change does not need to be upstreamed. An equivalent feature has already been merged into reth 1.5.0. This release is using to reth 1.1.5, thus requiring the fork. This fork does not need to be maintained post testnet 1 lifetime.

Somewhere in the PR description?
This needs to be documented and easily accessible if needed in the future. Once I resolve this thread, it will be buried forever.

Reply:
This is a hotfix for the latest outage we had. Coincidentally, we are working on upgrading to reth 1.5, so the change here will be supported out-of-the-box. But for Testent I, we ship with reth 1.1.5, so we need this fix in.